cluster/ci/config/prow/plugins.yaml

```diff
@@ -872,9 +875,10 @@ approve:
   - redhat-developer/devconsole-operator
   - redhat-developer/devconsole-api
   - redhat-developer/git-service
+  - redhat-developer/service-binding-operator
   require_self_approval: true
-  lgtm_acts_as_approve: true
-  ignore_review_state: false
+  lgtm_acts_as_approve: false
+  ignore_review_state: true
```

Comment:
@baijum what does this `ignore_review_state` mean?

Reply:
ignore_review_state causes the approve plugin to ignore the GitHub review state. otherwise, an APPROVE github review is equivalent to leaving an "/approve" message

Reply:
(IF and ONLY if the person approving the review has rights to `/approve`)